dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/CollectionRestRepository.java

```diff
@@ -139,4 +160,107 @@ public CollectionResource wrapResource(CollectionRest collection, String... rels
         return new CollectionResource(collection, utils, rels);
     }
 
+    @Override
+    @PreAuthorize("hasAuthority('ADMIN')")
```

Comment:
I suspect we need to log a bug ticket to refactor `createAndReturn()`.   

As `createAndReturn` only takes in a Context object, I'm not sure it can use the more appropriate `@PreAuthorize("hasPermission(#id, 'COMMUNITY', 'ADD')")` annotation...because it doesn't have a place to grab the parent Community ID from.

So, while I agree with @abollini, I think this particular line needs to stay as-is, until we can refactor `createAndReturn` to allow for fine grained permission checks (I'll create a ticket for it)

Reply:
Logged as a bug here: https://jira.duraspace.org/browse/DS-4153